src/Beta/Microsoft/Graph/ExternalConnectors/Model/External.php

```diff
@@ -115,7 +115,7 @@ public function setODataType($val)
     *
     * @return array The list of properties
     */
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
I think this would be better as `array`.

Also, our models are auto-generated using a different tool. Please see our [Contribution Guide](https://github.com/microsoftgraph/msgraph-sdk-php/blob/dev/CONTRIBUTING.md#how-the-library-is-built) and consider making this change on the code generator.

Reply:
@Ndiritu  i have changed to :array

but how to change your code generator, i did not know... may be you can post me a link, where i have to change it... ?

Reply:
@ihr-it-projekt sorry for the delay.

You can make the change to the template [here](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/dev/Templates/templates/PHP/Model/EntityType.php.tt#L285) & submit a PR to that repo.

With that, I'll ask that you remove the model updates from this PR.

Reply:
In case of any challenges, please let me know. Happy to help